_templates/bs/new/Demo.ejs.t

```diff
@@ -0,0 +1,5 @@
+---
+to: packages/<%= name %>/src/Demo.re
```

Comment:
Can we rename the file to the Pascal case of the name of the package?

Reply:
lerna alread do this in /lib :thinking: